alibi/explainers/permutation_importance.py

```diff
@@ -38,105 +38,125 @@ class Kind(str, Enum):
-        for documentation.
+    - ``'mean_squared_log_error'`` - Mean squared logarithmic error regression loss. \
+    See `sklearn.metrics.mean_squared_log_error`_ for documentation.
 
-        - ``'mean_squared_error'`` - Mean squared error regression loss. See `sklearn.metrics.mean_squared_error`_ \
-        for documentation.
+    - ``'mean_absolute_percentage_error'`` - Mean absolute percentage error (MAPE) regression loss. \
+    See `sklearn.metrics.mean_absolute_percentage_error`_ for documentation.
 
-        - ``'mean_squared_log_error'`` - Mean squared logarithmic error regression loss. \
-        See `sklearn.metrics.mean_squared_log_error`_ for documentation.
+    - ``'log_loss'`` - Log loss, aka logistic loss or cross-entropy loss. \
+    See `sklearn.metrics.log_loss`_ for documentation.
 
-        - ``'median_absolute_error'`` - Median absolute error regression loss. \
-        See `sklearn.metrics.median_absolute_error`_ for documentation.
+        .. _sklearn.metrics.mean_absolute_error:
+            https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_error.html#sklearn.metrics.mean_absolute_error
 
-        - ``'mean_absolute_percentage_error'`` - Mean absolute percentage error (MAPE) regression loss. \
-        See `sklearn.metrics.mean_absolute_percentage_error`_ for documentation.
+        .. _sklearn.metrics.mean_squared_error:
+           https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html#sklearn.metrics.mean_squared_error
 
-            .. _sklearn.metrics.max_error:
-                https://scikit-learn.org/stable/modules/generated/sklearn.metrics.max_error.html#sklearn.metrics.max_error
+        .. _sklearn.metrics.mean_squared_log_error:
+            https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_log_error.html#sklearn.metrics.mean_squared_log_error
 
-            .. _sklearn.metrics.mean_absolute_error:
-                https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_error.html#sklearn.metrics.mean_absolute_error
+        .. _sklearn.metrics.mean_absolute_percentage_error:
+            https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_absolute_percentage_error.html#sklearn.metrics.mean_absolute_percentage_error
 
-            .. _sklearn.metrics.mean_squared_error:
-               https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_error.html#sklearn.metrics.mean_squared_error
+        .. _sklearn.metrics.log_loss:
+            https://scikit-learn.org/stable/modules/generated/sklearn.metrics.log_loss.html#sklearn.metrics.log_loss
+"""
 
-            .. _sklearn.metrics.mean_squared_log_error:
-                https://scikit-learn.org/stable/modules/generated/sklearn.metrics.mean_squared_log_error.html#sklearn.metrics.mean_squared_log_error
+LossFnsValues = Literal[
```

Comment:
Nitpick - I would stick to singular for type definitions and also change it to `Name` from `Value` as it's more natural, i.e. `LossFnName` and `ScoreFnName` respectively.